src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -669,4 +672,27 @@ public Response getProjectsWithoutDescendantsOf(
             }
         }
     }
+
+    @GET
```

Comment:
This endpoint should have its own API resource instead. There are multiple other cases where event tokens could be returned, and checking whether a given token is being processed will always use the same logic.

Thinking out loud, we may even want to allow users to manually trigger certain events via REST API in the future. With this in mind, how about we move this to `/api/v1/event/token/{uuid}` instead?